README.md

```diff
@@ -33,6 +33,7 @@ Include the following dependencies in your app's build.gradle :
 ```
 def barricadeVersion = "0.0.1" // Get the latest version from tags
 dependencies {
+    implementation ("om.mutualmobile:barricade-annotations:0.1.1")
```

Comment:
Can we fix the spelling here i.e. `com.mutualmobile:barricade-annotations:0.1.1` instead of `om.mutualmobile:barricade-annotations:0.1.1`?